base/buildspec.yml

```diff
@@ -26,8 +26,8 @@ images:
     image_size_baseline: 1786
     device_type: &DEVICE_TYPE cpu
     python_version: &DOCKER_PYTHON_VERSION py3
-    tag_python_version: &TAG_PYTHON_VERSION py37
-    os_version: &OS_VERSION ubuntu18.04
+    tag_python_version: &TAG_PYTHON_VERSION py38
```

Comment:
I wonder if we should consider moving the py and ubuntu version to a separate common location (at least on a container level) so that they can be referenced in here and in the dockerfile.

Reply:
Perhaps that could be valuable, especially if we were to start building images with different versions. For now though, this quick change will get the tags to accurately reflect the versions that are already being built.